index.js

```diff
@@ -67,7 +67,7 @@ module.exports = options => {
 				insert(result, 'alias', alias, key);
 			});
 
-			if (props.default) {
+			if (Object.prototype.hasOwnProperty.call(props, 'default')) {
```

Comment:
Oh right. Could you use `{}.hasOwnProperty.call(props, 'default')` instead, in that case?